### ISSUE TITLE
Windows 10 mobile: touch locations are offseted

### ISSUE THREAD

**Issue by KakCAT** [MonoGame/MonoGame#4099]
Hi,

creating and executing a MonoGame project in Windows 10 mobile (with templates from the latest build) makes the touch locations offseted, probably due to the top bar. Problem appears both in portrait and landscape (offset amount changes though)

Minimal sample showing the problem: http://pastebin.com/pSDQR1U5

**Comment by KakCAT** [comment]
Hi,

I think I located the problem but I'm afraid to break something in the process of fixing W10mobile in the multiple platforms MonoGame supports added to my genetic inhability to use github :) . Could somebody please have a look at it?



    UIElement savedInputElement;
		
    public InputEvents(CoreWindow window, UIElement inputElement, TouchQueue touchQueue)
    {
        savedInputElement=inputElement;
        ....


and then 

    var pointerPoint = args.GetCurrentPoint(savedInputElement);

in the 4 places it's used.


GetCurrentPoint is currently called with null, which I suppose in WP10 means the 'root' of the screen, but for some reason this must be handled differently than in WP8.1, because WP8.1 seems to work correctly without that parameter...

Any ideas?

**Comment by tomspilman** [comment]
@KakCAT 

The `savedInputElement` thing was just a way to support both XAML and non-XAML versions of Windows 8 apps.  If you were running a XAML app the UIElement would be non-null and all input would be relative to it.  This included letting other XAML controls overlaying the game screen capture input events.

In Win10 all apps are XAML apps and the `SwapChainPanel` is passed in as the `UIElement` to `InputEvents`.

**Comment by KakCAT** [comment]
Hi Tom,

I'm sorry, my message was not clear enough (I've edited the second message for clarity)

The code I posted is not actual MonoGame code but a fix I think fixes the problem for WP10.

The problem is that in Windows 10 (mobile), you must pass the InputElement to the GetCurrentPoint function, otherwise the touches are offseted a certain amount determined by the top windows phone bar. Right now 'null' is being passed to GetCurrentPoint. 

My doubt is that I don't know how the modification will affect other platforms
